Agreements/Internal/S_D.txt

```diff
@@ -21,6 +21,8 @@ COPX:EGFF:*:*:*:*:London AC Worthing:London AC Dover:29000:*:^BPK
 COPX:EGFH:*:*:*:*:London AC Worthing:London AC Dover:29000:*:^BPK
 COPX:EGSY:*:*:*:*:London AC Worthing:London AC Dover:29000:*:^BPK
 
+COPX:EGLF:*:*:CLN:*:London AC Worthing:London AC Dover:21000:*:CLN
```

Comment:
DIKVU please

Reply:
Just to clarify you want FIX = DIKVU, FIXAFTER = CLN?

Reply:
The traffic between AC S and AC Dover never goes via CLN

Reply:
```
Via CLN
TC SW->TC VATON FL210 level OCK
TC VATON -> AC Dover FL210
AC Dover -> S12 FL230
```
I thought TC VATON was always covered by AC S?

Reply:
Ah apologies. Forgetting the rules myself. 

In which case the agreement in TC VATON should be at BRAIN for those via CLN to S12
And then it's at DIKVU for TC VATON to AC Dover